src/types/Transaction.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ *
+ */
+
+import { PaymentMethod } from "./PaymentMethod";
+
+/**
+ *
+ */
+export interface ITransaction {
+    readonly method: PaymentMethod;
+    readonly fee: number;
```

Comment:
be more restrictive with types, always put constraint wherever you can.
ie. we have a `NonNegativeNumber` type in [italia-ts-commons](https://teamdigitale.github.io/italia-ts-commons/) and a [`DateFromString`](https://github.com/teamdigitale/italia-ts-commons/blob/master/src/dates.ts) type.

Moreover, when you have booleans in a type, consider to split it according to the boolean value. ie in this case you may have a type `NewTransaction` (when isNew = true) so you can pass them along other methods being assured that it is what they expects. otherwise you have to do a check for `isNew` in every method deep down the call stack.